.github/CODEOWNERS

```diff
@@ -2,7 +2,7 @@
 # the repo. Unless a later match takes precedence.
 # By convention in @tablelandnetwork repos, the first listed
 # CODEOWNER is the repo DRI (directly responsible individual)
-*           @jsign @brunocalza @asutula
+*           @jsign @brunocalza
```

Comment:
Removing Aaron from general to avoid pinging him with PR reviews for most PRs. We still have him in the `pkg/client` which make sense.